madgiraffestudios/src/components/Navigation/Navigation.css

```diff
@@ -0,0 +1,4 @@
+.navbar-expand-lg .navbar-collapse {
+    font-family: 'Poppins' !important;
+    
+  }
```

Comment:
please remove, poppins should be for the entire project - this way we don't need to set it here for navigation only

Reply:
addressed